content/community/raising-issues/Types of Issue Labels.adoc

```diff
@@ -0,0 +1,57 @@
+---
+Title: Types of Issue Labels
+Weight: 20
+---
+
+Types of Issue Labels
```

Comment:
Let me suggest a few improvements to this table structure and formatting:

Let's avoid these `[gray-background]` labels and use Asciidoc's inbuilt formatting, and our stylesheets (which ensure consistency):
See https://asciidoctor.org/docs/user-manual/#cols-format , and scroll down to where you find `style component` in the text. We just need to specify "emphasis" for the first column.

Then, I suggest removing the middle column altogether, and instead make the labels in the first column links to those searches, like this:
[Assessed](https://github.com/salesagility/SuiteCRM/pulls?q=is%3Aopen+is%3Apr+label%3AAssessed)

Use also Asciidoc's column size specifiers to make the first column narrower, for example `[cols="30,70"]`

Reply:
@camy did you see all the other suggestions in this comment?

Reply:
I just had to re-learn this for another PR, so I can be more specific here.

A nice way to format the table is this:

`[cols="30h,70", options="header"]`

This will take care of the widths, and also of adding emphasis to the top row and the first column. So all the other emphasis in specific cells can be removed. 

The second column just goes into the first column as a link target, for example:

```
|https://github.com/salesagility/SuiteCRM/labels/Low%20Priority[Low priority^]
|Low impact (e.g. visual only, typos, alignments).
```

Note that all the links seem to be broken, they all point to `bug` label...